backend/src/routes/results.js

```diff
@@ -1,6 +1,20 @@
 import express from 'express';
+import Result from '../mongo/models/Result';
 const router = express.Router();
 
-// TODO: Handlers...
+// GET: Get results
+router.get('/results/:id', async (req, res) => {
+  const result = new Result({
+    sessionId: req.params.id,
+    restaurant: req.result.restaurant,
+  });
+
+  try {
+    const newResult = await result.save();
+    res.status(200).json(newResult);
+  } catch (error) {
+    res.status(500).json({message: error.message});
+  }
+});
```

Comment:
Since this is GET request, you should be returning the results document in the database of the specified id. But here you are creating a results object in the database